packages/dd-trace/src/appsec/addresses.js

```diff
@@ -20,5 +20,7 @@ module.exports = {
   HTTP_CLIENT_IP: 'http.client_ip',
 
   USER_ID: 'usr.id',
-  WAF_CONTEXT_PROCESSOR: 'waf.context.processor'
+  WAF_CONTEXT_PROCESSOR: 'waf.context.processor',
+
+  URL_ACCESS: 'server.io.net.url'
```

Comment:
i'm still not convinced about the naming, why can't this follow the current nomenclature like: `HTTP_OUTGOING_URL` (yes i've realized line 18 is wrong as it should be `HTTP_INCOMING_RESPONSE_BODY` instead of `HTTP_OUTGOING_BODY`)

Reply:
The main reason why this var is not `HTTP_OUTGOING_URL` is that `HTTP_OUTGOING_` is already used for response things.

I can modify `HTTP_OUTGOING_BODY` to `HTTP_INCOMING_RESPONSE_BODY` and this to `HTTP_OUTGOING_URL`

Reply:
You can tell we fucked up the nomenclature because we both have `HTTP_INCOMING_RESPONSE_CODE` and `HTTP_INCOMING_RESPONSE_HEADERS` which implies, HTTP_INCOMING means a server request/response lifecycle. If I remember correctly this is what I was going for when I first wrote this file.
WDYT about fixing the nomenclature a little bit and going for my suggestion ?